ui/src/message_popup/linear_session/linear_session.jsx

```diff
@@ -57,9 +69,13 @@ export default React.createClass({
   renderContent() {
     const {questions} = this.props;
     const {responses} = this.state;
-    if (responses.length >= questions.length) return this.props.summaryEl(questions, responses);
 
-    const question = questions[responses.length];
+    if (this.props.getNextQuestion === undefined || this.props.getNextQuestion(questions, responses) === null) {
```

Comment:
Even with `getDefaultProps`, flow still throws an error if I don't check that the function is not undefined as shown in the screenshot below. I only added the check for getNextQuestion in order to eliminate that error from flow. That error actually causes the build to fail the automated tests run on github. I've found a [github issue](https://github.com/facebook/flow/issues/1660) related to this but don't yet know how to work around it without the explicit check. Perhaps we can look at this together

<img width="1099" alt="screenshot 2017-03-30 19 38 52" src="https://cloud.githubusercontent.com/assets/9402134/24530662/60cfe902-1581-11e7-8aba-fe803a6272a9.png">